src/test/java/org/jpeek/MetricsTest.java

```diff
@@ -146,7 +145,29 @@
             new Object[] {"OverloadMethods", "LCOM3", 0.25d},
             new Object[] {"TwoCommonAttributes", "LCOM3", 1.0d},
             new Object[] {"WithoutAttributes", "LCOM3", 0.0d},
+            new Object[] {"Bar", "PCC", 0.5d},
```

Comment:
@Serranya By default in Java, floating point literals are double. So, the `d` in the literal is redundant. You can check it [here](https://stackoverflow.com/questions/41289696/floating-point-literal-floating-literal-double-literal/41289715).

Reply:
@fabriciofx I wanted to stay consistent with the rest of the formatting. I think we should either keep the `d` suffix in the added test cases, or also remove the `d` in all the other test cases in this file.

wdyt?